Cubical/Categories/Displayed/Constructions/Graph.agda

```diff
@@ -0,0 +1,75 @@
+{-
+  The graph category of a profunctor viewed as a displayed category
+  over the product.
+-}
```

Comment:
As you say, this is different from the ordinary Grothendieck construction, so I'm not sure what you are suggesting. If anything this is the most general form of the Grothendieck construction.

Reply:
Yeah, this is probably too late, but I think you could have achieved the main result using instead the usual Grothendieck construction to avoid adding new things.

In any case, IMO naming this `Graph` is misleading, maybe 2-Sided-Grothendieck or something similar? The nLab page may also want some changes. (no one would talk about the graph of a relation to mean the subset of `A×B` because that *is* the relation)

Reply:
In double category theory, this construction is called the "tabulator" (https://ncatlab.org/nlab/show/tabulator) so maybe that?

Reply:
We're willing to change the main result if it's cleaner (we've already changed it several times for that reason)